src/main/java/io/confluent/idesidecar/restapi/credentials/Credentials.java

```diff
@@ -75,6 +92,16 @@ default boolean isOauth2() {
     return type() == Type.OAUTH2;
   }
 
+  /**
+   * Return true if this is an SCRAM credentials object.
+   *
+   * @return true if {@link #type()} equals {@link Type#SRCAM}
```

Comment:
nit, for the sake of completeness:

```suggestion
   * @return true if {@link #type()} equals {@link Type#SRCAM}, return false otherwise
```